2021-06-30 15:38:28,958 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2021-06-30 15:38:28,958 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from /home/tommi/Dokumente/Studium/Wiztro/bot/insta/venv/lib/python3.6/site-packages/instabot/bot/bot.py
2021-06-30 15:38:28,958 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2021-06-30 15:38:29,113 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2021-06-30 15:38:29,113 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2021-06-30 15:38:29,113 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2021-06-30 15:38:29,113 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
